index.js

```diff
@@ -0,0 +1,131 @@
+/* eslint-disable import/prefer-default-export */
+/* eslint-disable max-classes-per-file */
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-use-before-define */
+
```

Comment:
- [ ] OPTIONAL
You have gone far ahead of this milestone! Kindly ensure to work within the requirements of each milestone/activity

Reply:
kindly explain more on this cause whatever I did was inline with the milestone requirement.

Reply:
@misheck12 
In this project, I noticed that you've created a full website, which was actually supposed to be implemented when you reach milestone 4 instead!